app/views/sephora_style_guide/home/_tabs.slim

```diff
@@ -0,0 +1,79 @@
+article#tabs
+  h2.h2 Tabs
+
+  section
+    h3.h3 Default
+    .tabs
```

Comment:
@zamakkat yep, when window size is reduce and tabs should turn to scrollable tabs but also we don't want to show scrollbar show I used `container` to fix such kind of requirements .